core/src/main/scala/cats/Reducible.scala

```diff
@@ -60,6 +60,12 @@ import simulacrum.typeclass
   def reduceLeftTo[A, B](fa: F[A])(f: A => B)(g: (B, A) => B): B
 
   /**
+    * Monadic variant of [[reduceLeftTo]]
+    */
+  def reduceLeftM[G[_], A, B](fa: F[A])(f: A => G[B])(g: (B, A) => G[B])(implicit G: Monad[G]): G[B] =
```

Comment:
And since another change has been suggested I feel less guilty about this one: can you shift the asterisks one space to the left to be javadoc style like other comments in the file? :)